test/RachisTests/RavenDB_14348.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using FastTests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace RachisTests
+{
+    public class RavenDB_14348 : RavenTestBase
+    {
+        public RavenDB_14348(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ShouldWork()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var task = Task.Run(async () =>
```

Comment:
This task "leaks"

Reply:
it is stopped when store gets disposed
https://github.com/ravendb/ravendb/blob/v4.2/test/RachisTests/SubscriptionFailoverWIthWaitingChains.cs#L310